uptime-kuma/templates/service.yaml

```diff
@@ -4,6 +4,12 @@ metadata:
   name: {{ include "uptime-kuma.fullname" . }}
   labels:
     {{- include "uptime-kuma.labels" . | nindent 4 }}
+    {{- with .Values.service.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
please add this to the readme too :wink:

*I noticed that I missed adding some too :S if you notice some could you please add them to the readme too? otherwise I will tackle that on the weekend.

Reply:
I will go through them, i noticed some missing too but wasn't sure I saw that right, ill add those as my boyscout to the 1.1.0 release :D